core/os/input_event.h

```diff
@@ -44,7 +44,7 @@
  * The events are pretty obvious.
  */
 
-enum {
+enum ButtonList {
```

Comment:
I know you chose this naming because of the already existing `KeyList` enum, but as these appear in the docs I think something like `MouseButton` and `JoypadButton` would be clearer. That'd also mean splitting the enum for joy buttons and joy axes.